src/test/ui/parser/issues/issue-66357-unexpected-unreachable.rs

```diff
@@ -11,4 +11,4 @@
 
 fn f() { |[](* }
 //~^ ERROR expected one of `,` or `:`, found `(`
-//~| ERROR expected one of `&`, `(`, `)`, `-`, `...`, `..=`, `..`, `[`, `_`, `box`, `mut`, `ref`, `|`, identifier, or path, found `*`
+//~| ERROR expected one of `&`, `(`, `)`, `-`, `...`, `..=`, `..`, `/*«*/`, `[`, `_`, `box`, `mut`, `ref`, `|`, identifier, or path, found `*`
```

Comment:
The `/*«*/` thing shouldn't be shown in user-visible diagnostics.
Or maybe should be listed as "invisible parenthesis".

Reply:
I'm inclined to leave is as `/*«*/`. That can show up in pretty-printed output, why not use it in error message too, for consistency?

Reply:
Pretty-printed output is not commonly seen by users, unlike these errors.
But we can wait until someone actually submits a "what is this thing, it frightens me" issue.

Reply:
I've changed it to text, like this:
```
ERROR expected unsuffixed literal or identifier, found `/*start of expr expansion*/`
```